cmd/state-offline-installer/offline-installer.go

```diff
@@ -67,28 +69,26 @@ func run() error {
 			return nil
 		},
 	)
-	if isZip(params.backpackZipFile) {
-		cmd.AddChildren(
-			captain.NewCommand(
-				cmdInstall,
-				"Doing offline installation",
-				"Do an offline installation",
-				p, nil,
-				[]*captain.Argument{
-					{
-						Name:        "path",
-						Description: "Install into target directory <path>",
-						Value:       &params.path,
-						Required:    true,
-					},
-				},
-				func(ccmd *captain.Command, args []string) error {
-					logging.Debug("Running CmdInstall")
-					return runInstall(out, params)
+	cmd.AddChildren(
+		captain.NewCommand(
+			cmdInstall,
```

Comment:
It's a little awkward that you have to run `./installer install` rather than just `./installer`. Why not get rid of the subcommand and run off of the main command?